### PR TITLE
fix: motion tab columns spacing

### PR DESCRIPTION
**Pull request by jnm2377**
Rel #1679 

**Changed**

- fix mobile column spacing in motion overview page tabs

### CODE REVIEW

File: src/pages/guidelines/motion/overview.mdx [hunk 1]
```diff
@@ -50,13 +50,13 @@ user’s attention and offer a rhythmic break to the productive experience.
...
 <Column colSm={6} colMd={4} colLg={6}>
-<p>Productive</p>
+<p className="tab--columns_label"> Productive </p>
 
 ![Curve graphs representing differences between productive and expressive animation curves](images/Motion_overview_Curves_productive.svg)
 
 </Column>
 <Column colSm={6} colMd={4} colLg={6}>
-<p>Expressive</p>
+<p className="tab--columns_label"> Expressive </p>
```
Comment (emyarod) [inline src/pages/guidelines/motion/overview.mdx:59]:
```suggestion
<p className="tab--columns_label">Expressive</p>
```

is this intentional? I don't think this whitespace changes the text

**Review by emyarod — APPROVED** [review]
looks good to me pending visual review

File: src/pages/guidelines/motion/overview.mdx (hunk 1 above)
Comment (jnm2377) [reply src/pages/guidelines/motion/overview.mdx:59]: No, that wasn't intentional

**Review by aledavila — APPROVED** [review]

**Comment by jnm2377** [comment]
@jeanservaas is this ok?

**Pull request merged**